present/slideshow.py

```diff
@@ -112,7 +143,7 @@ def get_effects(self, slide):
         pad = 2
         for e in elements:
             if e.type == "code":
-                effects.extend(_code(self.screen, e, row))
+                effects.append((e, row))
```

Comment:
If we have a `code` element, we need to know which row it should appear on, but we don't want it to be an Effect since we will be using `screen.paint()`.